### PR TITLE
Setup VH for Frontdoor and backend services

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7222
https://tools.hmcts.net/jira/browse/VIH-7223
https://tools.hmcts.net/jira/browse/VIH-7224
https://tools.hmcts.net/jira/browse/VIH-7225
https://tools.hmcts.net/jira/browse/VIH-7226

### Change description ###
Add VH Frontdoor Config and Backend APIs for all services and all environments

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: environments/demo/demo.tfvars [hunk 1]
```diff
@@ -14,6 +14,34 @@ frontends = [
     custom_domain  = "pip-frontend.demo.platform.hmcts.net"
     backend_domain = ["firewall-prod-int-palo-sdsdemo.uksouth.cloudapp.azure.com"]
 
+    disabled_rules = {}
+  },
+  {
+    name           = "vh-test-web"
```
Comment (timja) [inline environments/demo/demo.tfvars:20]: don't expect this setup in demo to actually work, it's not setup and not planned like this currently

File: environments/prod/backend_lb_config.yaml [hunk 2]
```diff
@@ -29,3 +30,25 @@ gateways:
       - product: pip
         component: account-management
         ssl_enabled: true
+    # Video Hearings
+      - product: vh
```
Comment (timja) [inline environments/prod/backend_lb_config.yaml:34]: do you need to add to prod now? is the application there and running?

you're welcome to add it but just want to discourage noise from failing health probes if the application is not there

**Review by timja — APPROVED** [review]

File: environments/prod/backend_lb_config.yaml (hunk 2 above)
Comment (chris-pateman) [reply environments/prod/backend_lb_config.yaml:34]: They are looking to do production cut over March so we are setting all environments up.

File: environments/demo/demo.tfvars (hunk 1 above)
Comment (chris-pateman) [reply environments/demo/demo.tfvars]: Demo has been removed.

**Pull request merged**